modules/user_profiles/user_profiles.libraries.yml

```diff
@@ -0,0 +1,4 @@
+duplicate:
+  version: 1.x
+  js:
+    js/duplicate.js: {}
```

Comment:
I think we're not using this JavaScript anymore now that the merge is happening automatically?